app.py

```diff
@@ -1,19 +1,59 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import requests as Requests
+import secrets
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    item_list = session.get_items()
+    item_list = []
+    list_id = '5f56323626c33d81cd98b386'
+    r = Requests.get('https://api.trello.com/1/lists/{}/cards?key={}&token={}'.format(list_id, secrets.KEY, secrets.TOKEN))
```

Comment:
Python has a neater way of doing string interpolation:
```suggestion
    r = Requests.get(f'https://api.trello.com/1/lists/{list_id}/cards?key={secrets.KEY}&token={secrets.TOKEN}')
```